test/turbopack-tests-manifest.json

```diff
@@ -888,9 +890,7 @@
       "ReactRefreshLogBox app default Module not found missing global CSS",
       "ReactRefreshLogBox app default Node.js builtins"
     ],
-    "flakey": [
-      "ReactRefreshLogBox app turbo Module not found missing global CSS"
```

Comment:
You also need to remove that from the skip list in `build-turbopack-manifest.js`